.bldr.toml

```diff
@@ -273,6 +273,10 @@ plan_path = "erlang16"
 plan_path = "erlang18"
 [erlang19]
 plan_path = "erlang19"
+[erlang20]
+plan_path = "erlang20"
+[erlang21]
```

Comment:
I see you added erlang21 to the `.bldr.toml` files, but there isn't a corresponding `erlang21/plan.sh`.